myth/Api/Server/ApiController.php

```diff
@@ -186,41 +185,39 @@ public function __construct()
 			unset($file);
 		}
 
-	    parent::__construct();
```

Comment:
I have a notice Undefined property: `TestController::$input` and refers to `$this->input->server(...)`  which is called by `detectMethod()` that is before `parent: __construct();`